src/shared/pager.c

```diff
@@ -164,6 +164,26 @@ int pager_open(PagerFlags flags) {
                         _exit(EXIT_FAILURE);
                 }
 
+                /* People might invoke us from sudo, don't needlessly allow less to be a way to shell out
+                 * privileged stuff. (Of course, this just illustrates the misdesign of less, but still, it's
+                 * an easy lockdown, so let's do it.) */
```

Comment:
I don't think this comment is useful. While the whole situation is unfortunate, quite certainly it's not a misdesign of less. If anything, less is being used in a context where it shouldn't (without additional prep).

Reply:
oops, i wanted to write misdesign of sudo, not less